test_sets/SDL5_0/buttonSubscription.txt

```diff
@@ -0,0 +1,36 @@
+./test_scripts/API/ButtonSubscription/006_Unsubscribe_button_SDL_send_SubscribeButton_after_Unsubscribe_SUCCESS_timeout.lua
+./test_scripts/API/ButtonSubscription/007_Subscribe_Failure_with_Error_Code.lua
+./test_scripts/API/ButtonSubscription/008_Unsubscribe_Failure_with_Error_Code.lua
+./test_scripts/API/ButtonSubscription/009_SubscribeButton_INVALID_request_INVALID_DATA.lua
+./test_scripts/API/ButtonSubscription/010_UnsubscribeButton_INVALID_request_INVALID_DATA.lua
+./test_scripts/API/ButtonSubscription/011_SubscribeButton_twise_the_same_button_IGNORED.lua
+./test_scripts/API/ButtonSubscription/012_UnsunscribeButton_twise_the_same_button_IGNORED.lua
+./test_scripts/API/ButtonSubscription/013_SubscribeButton_without_capabilities.lua
+./test_scripts/API/ButtonSubscription/014_UnsubscribeButton_without_capabilities.lua
+./test_scripts/API/ButtonSubscription/015_SubscribeButton_OK_majorVersion_4.lua
+./test_scripts/API/ButtonSubscription/016_SubscribeButton_OK_majorVersion_4_PLAY_PAUSE_not_supported.lua
+./test_scripts/API/ButtonSubscription/017_SubscribeButton_PLAY_PAUSE_majorVersion_5.lua
+./test_scripts/API/ButtonSubscription/018_SubscribeButton_PLAY_PAUSE_majorVersion_5_PLAY_PAUSE_not_supported.lua
+./test_scripts/API/ButtonSubscription/019_UnsubscribeButton_OK_majorVersion_4.lua
+./test_scripts/API/ButtonSubscription/020_UnsubscribeButton_OK_majorVersion_4_PLAY_PAUSE_not_supported.lua
+./test_scripts/API/ButtonSubscription/021_UnsubscribeButton_PLAY_PAUSE_majorVersion_5.lua
+./test_scripts/API/ButtonSubscription/022_SubscribeButton_App1_App2_on_the_OK_button.lua
+./test_scripts/API/ButtonSubscription/023_SubscribeButton_App1_App2_on_the_different_buttons.lua
+./test_scripts/API/ButtonSubscription/024_UnsubscribeButton_App1_App2_on_the_OK_button.lua
+./test_scripts/API/ButtonSubscription/025_UnsubscribeButton_App1_App2_on_the_different_buttons.lua
+./test_scripts/API/ButtonSubscription/026_SubscribeButton_App1_App2_on_the_same_button.lua
+./test_scripts/API/ButtonSubscription/027_UnsubscribeButton_App1_App2_on_the_same_button.lua
+./test_scripts/API/ButtonSubscription/028_SubscribeButton_Resumption_Unexpected_disconect.lua
+./test_scripts/API/ButtonSubscription/029_SubscribeButton_IGN_OFF_ON.lua
+./test_scripts/API/ButtonSubscription/030_SubscribeButton_Unexpected_disconect_HMI_responds_ERROR_CODE.lua
+./test_scripts/API/ButtonSubscription/031_SubscribeButton_on_different_buttons_Unexpected_disconect.lua
+./test_scripts/API/ButtonSubscription/032_SubscribeButton_Unexpected_disconect_HMI_doesn_t_response_timeout.lua
+./test_scripts/API/ButtonSubscription/033_SubscribeButton_non_media_App.lua
+./test_scripts/API/ButtonSubscription/034_SubscribeButton_RPC_is_not_allowed_by_policy_DISALLOWED.lua
+./test_scripts/API/ButtonSubscription/035_UnsubscribeButton_RPC_is_not_allowed_by_policy_DISALLOWED.lua
+./test_scripts/API/ButtonSubscription/036_SubscribeButton_Custom_Button_send_event_to_Background.lua
```

Comment:
Please update test set with actual list of the scripts and rename test set to name of the feature branch, defined in feature JIRA task `button_subscription_response_from_hmi.lua` - this is the rule for naming of the test sets